tests/integration/defs/test_doc.py

```
@@ -0,0 +1,241 @@
+                               allow_redirects=True,
+                               verify=False,
+                               stream=True)
+            resp.close()
+        if resp.status_code == 404:
+            return False, url, line_num, "404 Not Found"
+        return True, url, line_num, f"HTTP {resp.status_code}"
+    except requests.exceptions.RequestException as e:
+        if "Connection" in str(e):
+            return True, url, line_num, "connection issue (transient)"
+        return False, url, line_num, str(e)
+    except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)